src/olympia/addons/tests/test_indexers.py

```diff
@@ -102,6 +117,43 @@ def test_extract_attributes(self):
         for field_name in self.simple_fields:
             assert extracted[field_name] == getattr(self.addon, field_name)
 
+        assert extracted['app'] == [FIREFOX.id]
```

Comment:
We should at least decide something and stick to it :) `category` has the same issue. I didn't change it because that'd be a backwards-incompatible change needing a re-index to address, and I didn't want to go through existing views using search just yet...